examples/location_builders/lib/main.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:beamer/beamer.dart';
+import 'package:flutter/material.dart';
+
+import 'location_builders.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  final routerDelegate = BeamerRouterDelegate(
+    // As locationBuilder you can either use SimpleLocationBuilder or BeamerLocationBuilder.
+    // They are interchangeable, depending on personal taste (in this case).
+    // You can find both implementations in the location_builders.dart file.
+    //
+    // OPTION A: SimpleLocationBuilder
+    locationBuilder: simpleLocationBuilder,
+    //
+    // OPTION B: BeamerLocationBuilder
+    //locationBuilder: beamerLocationBuilder,
+  );
```

Comment:
Add option C:
```dart
locationBuilder: (state) => BooksLocation(state)
```

Reply:
Okay 👍 I will add that.

Reply:
And how should we name the variable for option C?